app/api/business/brief/brief_edit_business.py

```diff
@@ -187,6 +233,25 @@ def edit_seller_selector(brief, sellers_to_invite):
     return brief
 
 
+def edit_attachments(brief, documents):
```

Comment:
just looking at this, is there a way to write these functions as mutable?

Reply:
I've removed the return statements from these functions.